refs/types.go

```diff
@@ -35,11 +35,13 @@ type SignatureScheme uint32
 
 //nolint:revive
 const (
-	UnspecifiedScheme SignatureScheme = iota
-	ECDSA_SHA512
+	ECDSA_SHA512 SignatureScheme = iota
 	ECDSA_RFC6979_SHA256
 )
 
+// Deprecated: (v2.12.0) use ECDSA_SHA512 instead.
+const UnspecifiedScheme = ECDSA_SHA512
```

Comment:
Maybe call it default scheme and return default value down below? 
```suggestion
const DefaultSceme = ECDSA_SHA512
```

Reply:
We can't just rename it because of compatibility issues. Do you think we need to create additional `const DefaultScheme`?

Reply:
I don't think we need t create additional const. I think we can update release of api-go the same way as we going to update api.

Reply:
I got it, then I'll remove all deprecated elements.